CHANGELOG/CHANGELOG-3.5.md

```diff
@@ -3,7 +3,7 @@
 Previous change logs can be found at [CHANGELOG-3.4](https://github.com/etcd-io/etcd/blob/main/CHANGELOG/CHANGELOG-3.4.md).
 <hr>
 
-## v3.5.17 (TBC)
+## v3.5.17 (2024-11-12)
```

Comment:
Thanks @ghouscht can you also please scaffold the new version, example here https://github.com/etcd-io/etcd/pull/18573/files

Reply:
done!